.editorconfig

```diff
@@ -7,7 +7,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[{*.json,.*rc,*.yml}]
+[{*.json,.*rc,*.yml,*.css}]
```

Comment:
the CSS files are all formatted with 2 spaces rather than tabs. so i added this to avoid editors adding tabs each time (the pre commit catches it but its still a pain)